backstopJsFix/runDocker.js

```diff
@@ -0,0 +1,41 @@
+const { spawn } = require('child_process');
```

Comment:
could we move this into `scripts/` folder?

Reply:
You mean the file? Yes off course I going to move it right now

Reply:
Done